shared/wallets/wallet/settings/container.js

```diff
@@ -36,7 +36,6 @@ const mapDispatchToProps = (dispatch, {routeProps, navigateUp, navigateAppend})
   },
   _refresh: () => {
     dispatch(WalletsGen.createLoadDisplayCurrencies())
-    dispatch(WalletsGen.createLoadDisplayCurrency({accountID: routeProps.get('accountID')}))
```

Comment:
I think we should keep this call so we know it's up to date when showing this screen.

Reply:
ok